modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/frontend-token-definition.json

```diff
@@ -522,7 +522,7 @@
 							"type": "String"
 						},
 						{
-							"defaultValue": "#272833",
+							"defaultValue": "#6b6c7e",
```

Comment:
We have this same colour in other places as `#6B6C7E`. Unless I'm mistaken, we're using upper case for HEX colours.

Reply:
There's actually a big mix of upper and lower case here... 😭 

This can become an issue when "grepping" for occurrences with case sensitive settings that we might fail to find the same colour because it uses different combination of upper and lower case letters.

@wincent, I thought we had a rule for this? This is of course a `json` file, so it won't apply here, but wondering if the rule is all lowercase now as seen in the `.scss` in this same PR.

In either case, we should have a single format here, I guess

Reply:
oh! that's true!

I'm going to fix this and the file!

Reply:
> @wincent, I thought we had a rule for this? This is of course a json file, so it won't apply here, but wondering if the rule is all lowercase now as seen in the .scss in this same PR.

You basically said it:

- We don't enforce anything like this inside JSON.
- "We" do enforce it in SCSS, by which I mean, Prettier enforces it (to lowercase).

I think we could pretty easily do this in JSON using liferay-npm-scripts, at least for the obvious case; ie:

- Strings exactly matching `/^[a-f0-9]{3,6}$/i`.

I mean, you might end up formatting something you don't meant to format (eg. a hashtag ... 🤦‍♂️) but we could always ignore such edge case if it ever came up.

Created https://github.com/liferay/liferay-npm-tools/issues/494 to track this.